jobs/archiver_syslog/spec

```diff
@@ -90,6 +91,16 @@ properties:
     description: Transport protocol to use
     default: "tcp"
 
+  logstash_ingestor.health.disable_post_start:
+    description: Allow node to run post-start script? (true / false)
+    default: false
```

Comment:
There are some inconsistency between property name and description.
`Disable = false` should mean `enable`. However, `Allow = false` should mean `disable`.
Could you please elaborate that?

Reply:
The shorter path for my team would be to update the description to something like `Skip the post-start script` - does that work for you

Reply:
Yes please

Reply:
Sorry for the lag - this has been updated now